assets/scripts/woocommerce.js

```diff
@@ -8,14 +8,21 @@ jQuery(document).ready(function ($) {
   $('.field--salutation select').bind('change.ssofact', function () {
     var prefix = this.id.split('_')[0];
     var isCompany = $(this).val() === 'Firma';
-    $('#' + prefix + '_company_field, #' + prefix + '_company_contact_field')
+    var input;
+    input = $('#' + prefix + '_company_field, #' + prefix + '_company_contact_field')
```

Comment:
What about...
```diff
- var input;
- var input = $('#' + prefix + '_company_field, #' + prefix + '_company_contact_field')
+ var input = $('#' + prefix + '_company_field, #' + prefix + '_company_contact_field')
```

Reply:
Yes, but the `var` cannot / should not be repeated for the second value assignment below:
https://github.com/netzstrategen/wordpress-ssofact/pull/47/files#diff-edf39c844ed41c0b6e2359c25ad23388R19

As the two code blocks are doing the same with only subtle differences I preferred a higher degree of comparability, so that it is easier to the human eye to spot the differences.